src/schema.test.ts

```diff
@@ -231,18 +231,18 @@ describe('invalid categories entries', () => {
 
 describe('invalid applications entries', () => {
   const category = {
-    id: 'exchanges',
+    id: 'social-impact',
     backgroundColor: '#DEF8EA',
     fontColor: '#1AB775',
   }
-  it('errors on empty applications array', () => {
+  it('errors on missing name', () => {
     const testDappObject = {
       categories: [category],
       applications: [
         {
-          id: 'ubeswap',
-          categoryId: 'exchanges',
-          url: 'https://app.ubeswap.org/',
+          id: 'gooddollar',
```

Comment:
nit: maybe we should use Uniswap instead? Pretty sure it will be listed for the long run.

Reply:
I went with GoodDollar as it's the top performing dapp so hopefully it's also listed for the long run🤞